quickwit/quickwit-opentelemetry/src/otlp/logs.rs

```diff
@@ -301,21 +305,27 @@ impl OtlpGrpcLogsService {
     fn parse_logs(
         request: ExportLogsServiceRequest,
         parent_span: RuntimeSpan,
-        index_id: IndexId,
     ) -> Result<ParsedLogRecords, Status> {
         let (log_records, mut num_parse_errors) = parse_otlp_logs(request)?;
         let num_log_records = log_records.len() as u64;
         let mut error_message = String::new();
 
-        let mut doc_batch = DocBatchBuilder::new(index_id).json_writer();
+        let mut doc_batch_builder = DocBatchV2Builder::default().json_writer();
```

Comment:
Getting a `JsonDocBatchV2Builder` from `DocBatchV2Builder::default().json_writer()` is not straightforward.

Why not having `let json_doc_batch_builder = JsonDocBatchV2Builder::default();`?

Reply:
This was a 1 to 1 port of the V1 `DocBatchBuilder`, but I agree it could be simpler.